### PR TITLE
Add temporary info on how to re-enable chat

### PR DESCRIPTION
**Pull request by micwoj92**
I believe this could become one of frequently asked questions, in my opinion it is best to add it for short span of ~2 to 4 months so users who don't update regularly will have chance to see it.

### CODE REVIEW

File: docs/next_steps/troubleshoot.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ alias game_overrides_once_c hud_reloadscheme
 
 The command will only run on the first time you spawn in a match.
 
+## Last update disabled my chat
+
+One of [recent TF2 updates](https://www.teamfortress.com/post.php?id=62459) introduced new commands to disable chat. mastercomfig was updated shortly after to accomodate to this change. In the result, the Very Low preset disables chat completely. If you are using Very Low preset or `messages=off` module, you can fix that by adding `messages=on` to your `modules.cfg` file.
```
[the remark this answers is not in the document]
Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: already fixed bro, saw that before you

File: docs/next_steps/troubleshoot.md [hunk 2]
```diff
@@ -40,6 +40,11 @@ alias game_overrides_once_c hud_reloadscheme
 
 The command will only run on the first time you spawn in a match.
 
+## Last update disabled my chat
```
Comment (mastercoms) [inline docs/next_steps/troubleshoot.md]: Make this more generic: "My chat is disabled"

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/next_steps/troubleshoot.md (hunk 2 above)
Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: done

File: docs/next_steps/troubleshoot.md [hunk 3]
```diff
@@ -40,6 +40,11 @@ alias game_overrides_once_c hud_reloadscheme
 
 The command will only run on the first time you spawn in a match.
 
+## My chat is disabled
+
+One of [recent TF2 updates](https://www.teamfortress.com/post.php?id=62459) introduced new commands to disable chat. mastercomfig was updated shortly after to accommodate to this change. In the result, the Very Low preset disables chat completely. If you are using Very Low preset or `messages=off` module, you can fix that by adding `messages=on` to your `modules.cfg` file. To disable chat the old way you can use `hud_saytext_time 0` command.
```
Comment (mastercoms) [inline docs/next_steps/troubleshoot.md]:
```suggestion
[A TF2 update](https://www.teamfortress.com/post.php?id=62459) introduced the ability to completely disable text chat. mastercomfig was updated shortly after to accommodate this change, with the Very Low preset disabling chat completely. If you are using Very Low preset or the `messages=off` module, you can re-enable chat by adding `messages=on` to your `modules.cfg` file. To disable chat the old way you can use the `hud_saytext_time 0` command.
```

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/next_steps/troubleshoot.md (hunk 3 above)
Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: Oh this is actually better. I was thinking only in scope of the recent update. And that it would be temporary.

Comment (micwoj92) [reply docs/next_steps/troubleshoot.md]: done

**Review by mastercoms — APPROVED** [review]

**Pull request merged**